internal/configs/parsing_helpers.go

```diff
@@ -225,6 +225,102 @@ func ParseSize(s string) (string, error) {
 	return "", errors.New("Invalid size string")
 }
 
+// ParsePortList ensures that the string is a comma-separated list of port numbers
+func ParsePortList(s string) ([]int, error) {
+	ports := make([]int, 0)
+	for _, value := range strings.Split(s, ",") {
+		port, err := parsePort(value)
+		if err != nil {
+			return nil, err
+		}
+		ports = append(ports, port)
+	}
+	return ports, nil
+}
+
+func parsePort(value string) (int, error) {
+	port, err := strconv.ParseInt(value, 10, 16)
+	if err != nil {
+		return 0, fmt.Errorf("Unable to parse port as integer: %s", err)
+	}
+
+	if port <= 0 {
+		return 0, fmt.Errorf("Port number should be greater than zero: %q", port)
+	}
+
+	return int(port), nil
```

Comment:
```suggestion
	return port, nil
```
I think this would be ok

Reply:
The compiler complains if I change that, because port is an `int64`. :(